services-js/access-boston/src/integration/change-password.testcafe.ts

```diff
@@ -1,7 +1,7 @@
 import { Selector } from 'testcafe';
```

Comment:
Thanks for the reminder to chat with you about the ifs and whens on bringing testcafe in for other apps!

Reply:
Yeah, lemme know any time!